package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "node_modules/.bin/_mocha  $(find . -name '*.spec.js')",
-    "test:lint": "node_modules/eslint/bin/eslint.js ./  || exit 0",
+    "test:lint": "eslint .",
```

Comment:
It's trivial to fix the lint errors though. I can do it as part of this PR if you want?